src/index.js

```diff
@@ -153,12 +170,162 @@ const IN_CALL_BUTTONS_ARRAY = [
+            cancelButtonElement.onclick = () => {
+                modalElement.style.visibility = "hidden"
+                firstNameElement.value = ""
+                lastNameElement.value = ""
+                emailElement.value = ""
+            }
+
+            const newContactTitleElement = document.createElement("p")
+            newContactTitleElement.innerHTML = "New Contact"
+            newContactTitleElement.className = "new-contact"
+
+            const doneButttonElement = document.createElement("button")
+            doneButttonElement.innerHTML = "Done"
+            doneButttonElement.className = "modal-controls"
+            doneButttonElement.onclick = () => {
+                modalElement.style.visibility = "hidden"
+                const newContact = {}
+                newContact.firstName = firstNameElement.value
+                newContact.lastName = lastNameElement.value
+                newContact.phoneNumber = phoneNumberElement.value
+                newContact.eMail = emailElement.value
+
+                CONTACTS.push(newContact)
+            }
+
+            const addPhotoElementWrapper = document.createElement("div")
+            addPhotoElementWrapper.id = "add-photo-element-wrapper"
+
+            const addPhotoElement = document.createElement("p")
+            addPhotoElement.innerHTML = "Add Photo"
+            addPhotoElement.id = "add-photo"
+            addPhotoElement.onclick = () => {
+                console.log("Select photo from Gallery")
+            }
+
+            const addDetailsElementWrapper = document.createElement("div")
+            addDetailsElementWrapper.id = "add-details-element-wrapper"
+
+            const firstNameElement = document.createElement("input")
+            firstNameElement.id = "first-name"
+            firstNameElement.className = "details"
+            firstNameElement.placeholder = "First name"
+            firstNameElement.setAttribute("type", "text")
+
+            const lastNameElement = document.createElement("input")
+            lastNameElement.id = "last-name"
+            lastNameElement.className = "details"
+            lastNameElement.placeholder = "Last name"
+            lastNameElement.setAttribute("type", "text")
+
+            const phoneNumberElement = document.createElement("input")
+            phoneNumberElement.id = "phone-number"
+            phoneNumberElement.className = "details"
+            phoneNumberElement.value = displayElement.innerHTML
+
+            const emailElement = document.createElement("input")
+            emailElement.id = "email"
+            emailElement.className = "details"
+            emailElement.placeholder = "E-mail"
+            emailElement.setAttribute("type", "email")
+
+            document.body.insertBefore(modalElement, appElement)
+            modalElement.appendChild(modalControlWrapperElement)
+            modalControlWrapperElement.appendChild(cancelButtonElement)
```

Comment:
247:249

Do you know that there is a method available for parent which allows to add multiple children at the same time?